### PR TITLE
Vet clinic database: query multiple tables - Day 3

### PR DESCRIPTION
**Pull request by hajnaloltyan**
## Vet clinic database: query multiple tables - Day 3

In this PR I:
* Added the 2 new tables for owners and species
* Inserted data into the new tables, and modified animals table
* Inserted species types, and owner informations
* Wrote required queries and took screenshots of the results

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @hajnaloltyan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Let's remove all unnecessary headers in the README file. Live Demo, Deployment, Install headers are not useful in this project. Kindly provide commands on how to run the commands on a terminal

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -16,3 +16,39 @@ VALUES
...
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+ALTER TABLE animals
+    ADD COLUMN species_id integer REFERENCES species(id),
+    ADD COLUMN owner_id integer REFERENCES owners(id);
```
Comment (Shedrack-Sunday) [inline data.sql:34]: -[ ] Good job with the attempted modification 👏 . However, this should go into the `SCHEMA.SQL` file as instructed in the project requirements. This is to ensure proper best practices in project structure 👍 
- Kindly complete the modifications as specified 👍 
### Instructions:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @hajnaloltyan          👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with SQL features ✔️
- Good Validations set ✔️ 
    

# Required Changes  🌵  

- [ ] You've done well, However, kindly check some key points notes I pointed out to meet the requirements before moving to the next project 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rotimiazeez — APPROVED** [review]
Hi @hajnaloltyan 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- ✔️ Required tables created
- ✔️ Required data inserted into tables
- ✔️ Screenshots with the query results added

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**